### ISSUE TITLE
[Platform Client] Populate the tags & attributes available for each field from the backend

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#2810]
### Description: 
- A flexible table of tags & attributes pulled from HXL standards.
- Each field has a set list of attributes and tags available (for example `Location` fields have #geo+lat, #geo+lon)
- Each attribute is available for a specific tag. Not all attributes match all tags
### Tasks
- Get the information from the API <-- new issue needed to set up the api-endpoint in client
- Add the tags and attributes for each field to a dropdown according to designs https://xd.adobe.com/view/2690f082-d88d-4788-5db7-b04c9474a404-50a1/screen/6ebb69f3-8d21-46b4-8770-1eeb55a8e866/Data-export-HDX-single-survey

### Add qa-checklist
*Assumes HDX-featureflag is on*
- Go to settings->data-export->HDX
- The table should contain:
- [ ] A list of available surveys and survey-fields 
- [x] After each field there should be two dropdowns, one for tags and one for attributes
- Select one field with the checkbox
- [x] The first dropdown should become active and be populated with the tags fetched from the api
- Select one tag
- [x] The second dropdown should become active and be populated with the attributes connected to that tag
- Select another tag
- [x] New attributes should appear in the second dropdown

**Comment by Erioldoesdesign** [comment]
Can I also add here that tags and attributes should display alphabetically if possible?

(of course if they're pulled in from HDX via the API in a certain order let's not make extra work for ourselves in order to force alphabetical listing)

**Comment by rowasc** [comment]
@Erioldoesdesign I'm pretty sure we will need to add them ourselves and have no good way to pull them from an endpoint (still waiting on my meeting with hdx to be sure) so this would be feasible in that scenario. I'll confirm after we meet

**Comment by Angamanga** [comment]
@rowasc Could you please review?

**Comment by Angamanga** [comment]
@Eve-wanderi Ready for QA :)
